[DOCS] Update ingest node pipelines doc

alisonelizabeth opened the pull request:
This PR updates the ingest node pipeline doc based on changes made to the UI in `7.9` and `7.10`. The bulk of the changes are related to the example, which showed the old processors JSON editor, instead of the new processors editor component.

/cc @jloleysens if there's anything else you think we should add.

alisonelizabeth commented on docs/management/ingest-pipelines/ingest-pipelines.asciidoc:
```diff
@@ -62,47 +62,63 @@ You also want to know where the request is coming from.
+Once complete, click *Update*.
+. Add 3 more processors with the following data:
++
+*Date*
```
There might be a better way to display this data 😄

debadair replied:
Yeah, especially the way that the table is rendered. I think I'd split up the steps differently, instead. Something like:

```
. Add a grok processor to parse the log message:

.. Click *Add a processor* and select the *Grok* processor type.
.. Set the field input to `message` and enter the following grok pattern.
.. Click *Update* to save the processor.

. Add processors to map the Date, GeopIP, and User agent fields:

.. Select the ***Field*** processor type.
.. Map the appropriate parsed value to each field:
+
--
* Date: `timestamp`
* GeoIp: `clientip`
* User agent: `agent`

For the Date field, you also need to specify the date format you want to use.
--
```

alisonelizabeth replied:
Thanks! I like this a lot better. I followed this pattern, but changed the copy in a few places where I felt it wasn't 100% clear.

debadair commented on docs/management/ingest-pipelines/ingest-pipelines.asciidoc:
```diff
@@ -62,47 +62,63 @@ You also want to know where the request is coming from.
+
+|===
++
+You now have defined 4 {ref}/ingest-processors.html[processors] that run sequentially:
 {ref}/grok-processor.html[grok], {ref}/date-processor.html[date],
 {ref}/geoip-processor.html[geoip], and {ref}/user-agent-processor.html[user_agent].
```
```suggestion
The four {ref}/ingest-processors.html[processors]will run sequentially:
{ref}/grok-processor.html[grok], {ref}/date-processor.html[date],
{ref}/geoip-processor.html[geoip], and {ref}/user-agent-processor.html[user_agent].
```

debadair commented on docs/management/ingest-pipelines/ingest-pipelines.asciidoc:
```diff
@@ -62,47 +62,63 @@ You also want to know where the request is coming from.
+You now have defined 4 {ref}/ingest-processors.html[processors] that run sequentially:
 {ref}/grok-processor.html[grok], {ref}/date-processor.html[date],
 {ref}/geoip-processor.html[geoip], and {ref}/user-agent-processor.html[user_agent].
 Your form should look similar to this:
```
I'd move this before the part about them running sequentially.

debadair commented on docs/management/ingest-pipelines/ingest-pipelines.asciidoc:
```diff
@@ -62,47 +62,63 @@ You also want to know where the request is coming from.
 image:management/ingest-pipelines/images/ingest-pipeline-processor.png["Processors for Ingest Node Pipelines"]
 
-. To verify that the pipeline gives the expected outcome, click *Test pipeline*.
+. To verify that the pipeline gives the expected outcome, click *Add documents*.
```
```suggestion
. To test the pipeline to verify that it produces the expected results, click *Add documents*.
```

debadair commented on docs/management/ingest-pipelines/ingest-pipelines.asciidoc:
```diff
@@ -62,47 +62,63 @@ You also want to know where the request is coming from.
+. To verify that the pipeline gives the expected outcome, click *Add documents*.
 
-. In the *Document* tab, provide the following sample document for testing:
+. In the *Documents* tab, provide the following sample document for testing:
```
```suggestion
. In the *Documents* tab, provide a sample document for testing:
```
```suggestion
. In the *Documents* tab, provide a sample document for testing:
```

debadair reviewed (commented):
Left a few suggestions--let me know if you have any questions!

alisonelizabeth commented:
Thanks @debadair for the helpful review! I addressed your feedback. Would you mind taking another look when you get a chance?

jloleysens reviewed (commented):
@alisonelizabeth I am happy with the proposed set of changes. I wonder if it might be worth mentioning something about the capability to re-order processors too - don't think this is a hard requirement though.

alisonelizabeth commented:
Thanks @jloleysens! I added a sentence about reordering - good idea! I also added back the processors JSON and mentioned the "Import processors" ability.

debadair commented on docs/management/ingest-pipelines/ingest-pipelines.asciidoc:
```diff
@@ -90,19 +119,16 @@ You also want to know where the request is coming from.
-+
-[role="screenshot"]
-image:management/ingest-pipelines/images/ingest-pipeline-processor.png["Processors for Ingest Node Pipelines"]
+{ref}/geoip-processor.html[geoip], and {ref}/user-agent-processor.html[user_agent]. You can reorder processors using the arrow icon next to each processor. Reordering is not required for this example.
```
I don't think the note about reordering not being required is necessary. 

```suggestion
{ref}/geoip-processor.html[geoip], and {ref}/user-agent-processor.html[user_agent]. You can reorder processors using the arrow icon next to each processor. 
```

debadair reviewed (approved):
Left one minor suggestion, but LGTM.

alisonelizabeth commented:
@elasticmachine merge upstream
